docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md

```diff
@@ -0,0 +1,208 @@
+# Cómo configurar tu proyecto Open Source
+
+Hay muchas cosas a considerar al configurar tu proyecto Open Source. En esta parte del curso, revisaremos los componentes clave que todo proyecto necesita para tener éxito.
```

Comment:
consider changing "Hay muchas cosas a considerar al configurar tu proyecto Open Source.

to: "Hay muchas cosas a considerar al configurar tu proyecto **de** Open Source."

and

consider changing:  "En esta parte del curso, revisaremos los componentes clave que todo proyecto necesita para tener éxito."

to: "En esta parte del curso, revisaremos los componentes clave que **todos los proyectos necesitan** para tener éxito.